product_docs/docs/biganimal/release/getting_started/creating_a_cluster/01_cluster_networking.mdx

```diff
@@ -4,59 +4,33 @@ title: Cluster networking architecture
-address that always routes to the leader of your cluster. Once
-assigned, this IP address does not change unless you change the
-networking configuration for your cluster.
+BigAnimal uses the following resources for making routing decisions and distributing requests: 
 
-Only one Azure Load Balancer per BigAnimal region is typically deployed in your Azure
-subscription. Subsequent public clusters add more
-IP addresses to the existing load balancer.
+### Standard SKU Load Balancer in Azure
 
-## Private cluster load balancing
+When a cluster is created with public network access, a load balancer is created and configured with a public IP address. Once assigned, this IP address does not change unless you change the networking configuration for your cluster. The load balancer always routes to the leader of your cluster. 
 
-When a cluster is created with private network access, an Azure
-Standard SKU Load Balancer is created and configured with an IP
-address that always routes to the leader of your cluster. Once
-assigned, this IP address does not change unless you change the
-networking configuration for your cluster.
+Only one load balancer is typically deployed in an Azure region. BigAnimal adds more IP addresses to the existing load balancer for subsequent clusters in the Azure region.
 
-This IP address is private to the VNet hosting your BigAnimal
-services. By default it is not routable from other networks, even in
-your Azure account. See [Setting up Azure infrastructure to connect to a private network cluster](../../using_cluster/02_connecting_your_cluster/#setting-up-azure-infrastructure-to-connect-to-a-private-network-cluster) for details and instructions
-on how to properly configure routing for private clusters.
+Every BigAnimal cluster, regardless of public or private networking status, is assigned a single DNS zone that maps to its exposed IP address, either public or private. When toggling between public and private, wait up to 120 seconds for DNS caches to flush.
 
-Only one Azure Internal Load Balancer per BigAnimal region is typically deployed in your
-Azure subscription. Subsequent private clusters add more IP addresses to the existing load balancer.
+Clusters can be changed from public to private and vice versa at any time. When this happens, the IP address previously assigned to the cluster is deallocated, a new one is assigned, and DNS is updated accordingly.
 
-## DNS
+### Amazon Network Load Balancer in AWS. 
+ 
+BigAnimal creates a new load balancer for each clusters and tags it with the ckuster ID in the following format: 
 
-Every BigAnimal cluster, regardless of public or private networking
-status, is assigned a single DNS zone that maps to its exposed IP
-address, either public or private. When toggling between public and
-private, wait up to 120 seconds for DNS caches to flush.
+service.k8s.aws/stack: default/*<cluster_ID>* (for example, service.k8s.aws/stack: default/p-c4j0jfcmp3af2ieok5eg).
```

Comment:
This is a Kubernetes/AWS internal name; we should be talking about public DNS here.

Reply:
Does this look OK. 

Because the load balancer IP address used in AWS is dynamic, ensure that your application uses the correct DNS name to access the network load balancer of a particular cluster. In your application's AWS account, select the **Load Balancers** service, search for the load balancer with cluster ID you want to access, and use the DNS name provided in the details section to access your cluster.

Reply:
No, this doesn't look right at all, haha. They should get the DNS name from the BigAnimal portal, not AWS.